.github/workflows/superset-helm-lint.yml

```diff
@@ -1,4 +1,4 @@
-name: Lint and Test Charts
+name: "Helm: lint and test charts"
```

Comment:
Nice improvement... I'll bet half the people that see this think it's eslint and dataviz charts :)

Reply:
It's been bugging me for a moment. Making the repo a better place, one line at a time.